Integrations/integration-VirusTotal.yml

```diff
@@ -218,13 +218,20 @@ script:
             md += 'MD5 / SHA1 / SHA256: **' + r[i].md5 + ' / ' + r[i].sha1 + ' / ' + r[i].sha256 + '**\n';
             if (longFormat === 'true' && r[i].scans) {
                 scansTable = createScansTable(r[i].scans);
-                md += tableToMarkdown('Scans', scansTable)
+                md += tableToMarkdown('Scans', scansTable);
                 if (ec[outputPaths.file]){
                     scans_ec = {
                         Scans: scansTable,
                         ScanID: r[i].scan_id
+                    };
+                    if (typeof ec[outputPaths.file][0] === 'object') {
+                        ec[outputPaths.file][0].VirusTotal = scans_ec;
+                    } else {
```

Comment:
in which case `typeof ec[outputPaths.file][0] === 'object')` will not be object?
```
scans_ec = {
                        Scans: scansTable,
                        ScanID: r[i].scan_id
                    };
```
it seems always object